### DISCUSSION TITLE
バッジをマイページに表示する

### DISCUSSION THREAD

**Discussion by hellowork-mhlw** [increments/qiita-discussions#470] [要望・アイデア・不具合]
GitHubみたいにバッジをマイページに表示することで自己顕示欲が満たされていい思います。
Advent Calendar以外のバッジもあればいいかと思います
![image](https://user-images.githubusercontent.com/116468267/234548816-f4356824-3167-40ec-b2ed-586517f87859.png)
![image](https://user-images.githubusercontent.com/116468267/234549131-cd75e9a9-2bfc-49aa-8376-62055dfa0911.png)
https://qiita.com/badges

イメージ
![image](https://user-images.githubusercontent.com/116468267/234559059-c9773657-6a6b-402a-a9cc-ea903fd2043f.png)

**Comment by yoshi389111** [comment]
通りすがりの者ですが、過去、以下のようなディスカッションがありました。

#166 

> 現状、最近リリースしたバッジ機能なども含めて、実績が他者から見えない箇所が多いです
> プロフィールページのより良い姿を議論している最中でもありますので、議題として取り入れます！

#183 

> こちらのDiscussion (#166) でも似たようなご意見をいただいていまして、ユーザーページのアップデートを画策しています
> 上記の話と一緒に検討していきます！

ご参考まで。

**Comment by tanaka-asami415** [comment]
@yoshi389111 
回答いただきありがとうございます！

@hellowork-mhlw 
投稿ありがとうございます🙏
実現したいのは間違いないのですが、直近の開発優先度などから着手まで至っておらず申し訳ないです…🙇
